cmake/configs/default.cmake

```diff
@@ -41,7 +41,7 @@ hunter_config(Async++ VERSION 0.0.3-hunter)
 hunter_config(Avahi VERSION 0.6.31)
 hunter_config(Beast VERSION 1.0.0-b84-hunter-0)
 hunter_config(BZip2 VERSION 1.0.6-p3)
-if(MSVC OR MINGW)
+if(MINGW)
```

Comment:
> It should work fine as far as I can tell, the option exists since forever and only default behavior have changed. I've compiled boost example locally with my hunter master and forced version to 1.64 it seems to work. If you wish I can also re-run all tests with 1.64

Okay, this is great. Let's do it in two steps then, first check that we don't break old version, then do update to 1.66. 

Please revert changes in `cmake/configs/default.cmake` file and submit it later after merging.

Reply:
Done.